.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -19,7 +19,7 @@ Please complete the next sections or the issue will be closed.
 
 **OS**:
 
-<!-- MANDATORY! You can get it via `cat /etc/oss-release` for example -->
+<!-- MANDATORY! You can get it via `cat /etc/os-release` for example -->
```

Comment:
Hmm, I think I saw another PR on this a few days ago, but can’t find it now.

Reply:
Maybe you saw my comment inside #17727 about it?